lib/model.js

```diff
@@ -484,6 +484,7 @@ var validateIncludedElements = function(options, tableNames) {
         include.subQuery = false;
       } else {
         include.subQueryFilter = false;
+        include.subQuery = include.subQuery || include.hasParentRequired && (include.hasDuplicating || include.hasRequired);
```

Comment:
We need to add the include to the subquery if it's parent is in the subquery and itself or it's children is required.
Do we care about `hasDuplicated` here?